source/extensions/common/wasm/wavm/wavm.cc

```diff
@@ -276,8 +276,16 @@ bool Wavm::initialize(const std::string& code, absl::string_view name, bool allo
 
 void Wavm::start(Context* context) {
```

Comment:
@jplevyak unless I'm mistaken, this is never called in program, only in tests, which means that we neither initialize globals, nor call the start function (or `main()`), and starting program with optimized WASM module leads to a crash.

I think we've lost the call to `start()` during one of the refactors. Could you look into this? Thanks!